services/table-foreign-keys-analyzer.js

```diff
@@ -29,8 +30,11 @@ function TableForeignKeysAnalyzer(databaseConnection, config) {
             REFERENCED_TABLE_NAME AS foreign_table_name,
             REFERENCED_COLUMN_NAME AS foreign_column_name
           FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE
-          WHERE TABLE_SCHEMA = '${config.dbName}'
-            AND TABLE_NAME = '${table}';`;
+          WHERE TABLE_SCHEMA = :databaseName
+            AND TABLE_NAME = :table;`;
+        // NOTICE: `config.dbName` may not exists when using connection URL.
+        //         Use `dbConnectionUrl` as a fallback.
+        replacements.databaseName = config.dbName || config.dbConnectionUrl.split('/').pop();
```

Comment:
[Spec] This doesn't work with a custom schema `postgres://user:password@localhost:5432/my-db?schema=oops` And the schema can also contains slash !

Reply:
WHy not just retrieving the database name from sequelize 🤔  ?

Reply:
Good catch! Thanks!